server/views/pages/decision-evidence.njk

```diff
@@ -87,7 +87,7 @@
                         {
                         value: "false",
                         text: "No, it's not sensitive",
-                        checked: decisionEvidenceForm.isSensitive == false
+                        checked: (decisionEvidenceForm.isSensitive and decisionEvidenceForm.isSensitive == false)
```

Comment:
Personally I would put this logic in the `allocationStorageService`

Reply:
The trouble is that nunjucks treats null as false - so we have to code around that if we want a tri-state boolean, which isn't worth it right now